bip-0053.mediawiki

```diff
@@ -82,7 +82,7 @@ Notably this format does not commit to the height of the Merkle tree.
 Suppose a (valid) 64-byte transaction T is included in a block with the property that the second 32 bytes (which
 are less constrained than the first 32 bytes) are constructed so that they collide
 with the hash of some other fake, invalid transaction F. The attacker can fool the SPV client into believing that F
-was included in a Bitcoin block rather than T with 81 bits<ref>[https://github.com/Christewart/bips/blob/2024-12-20-64bytetxs/bip-XXXX/2-BitcoinMerkle.pdf An attacker who can do 81 bits of work (followed by another 40 bits of work, to
+was included in a Bitcoin block rather than T with 81 bits<ref>[https://github.com/Christewart/bips/blob/2024-12-20-64bytetxs/bip-0053/2-BitcoinMerkle.pdf An attacker who can do 81 bits of work (followed by another 40 bits of work, to
```

Comment:
```suggestion
was included in a Bitcoin block rather than T with 81 bits<ref>[bip-0053/2-BitcoinMerkle.pdf An attacker who can do 81 bits of work (followed by another 40 bits of work, to
```